app/models/flatirons/saas/concerns/subscriptable.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Flatirons
+  module Saas
+    module Concerns
+      # Subscriptable takes care of the relationship between any active record model and the subscriptions.
+      #
+      # ==Options
+      #
+      # TODO...
+      #
+      # == Examples
+      #
+      #   # get all subscriptions
+      #   User.find(1).subscriptions
+      #
```

Comment:
don't commit this

Reply:
@animista01 it is used to generate the documentation to our library.
We are using yard.
We are based on devise library.

Reply:
You can check the doc here: https://flatironsdevelopment.github.io/flatirons-saas/